hudi-utilities/src/main/java/org/apache/hudi/utilities/sources/processor/maxwell/MaxwellJsonKafkaSourcePostProcessor.java

```diff
@@ -111,9 +113,6 @@ public JavaRDD<String> process(JavaRDD<String> maxwellJsonRecords) {
 
       // filter out target databases and tables
       if (isTargetTable(database, table)) {
-
-        LOG.info(String.format("Maxwell source processor starts process table : %s.%s", database, table));
-
```

Comment:
There will be too much logs as it is logged in a map iterate, we'd better remove it.